product_assortment/tests/test_product_assortment.py

```diff
@@ -91,7 +91,24 @@ def test_product_assortment_view(self):
         included_product = self.env.ref("product.product_product_7")
         self.assortment.write({"whitelist_product_ids": [(4, included_product.id)]})
         res = self.assortment.show_products()
-        self.assertEqual(res["domain"], [("id", "in", [included_product.id])])
```

Comment:
This line has been changed to make the tests pass with the new orm behaviour.

Reply:
See comment https://github.com/OCA/product-attribute/pull/1864#issuecomment-2684047562 Why I think that, PR is more "complete".

Reply:
@PieterPaulussen Can you agree, on the fact that unit tests should not be altered on a functional level between versions to make them pass?

Additional unit test, as are added in both pr's are obviously ok.